Final_Project/Main.cpp

```diff
@@ -175,8 +175,6 @@ LRESULT CALLBACK WndProc(HWND hwnd, UINT Message,
 		case IDM_UPLOAD:
 			// handle file upload here
 			char ctrlFrame[1024]; //for test; to be removed
-			//generateCtrlFrame(ctrlFrame, 5); //for test; to be removed
-			receiveFrame(ctrlFrame); //for test; to be removed
 			sendCharacter(hwnd);
```

Comment:
when i comment out sendCharacter, the program runs fine and opens up file dialog. 

I tried with and without hypertrm connected and with it not commented out, the program freezes on clicking 'Upload'

everything else seems ok to me